Source/Cells/Text/FORMTextFieldCell.m

```diff
@@ -187,6 +187,7 @@ - (void)updateWithField:(FORMField *)field {
     self.textField.rawText         = [self rawTextForField:field];
     self.textField.info            = field.info;
     self.textField.styles          = field.styles;
+    self.textField.placeholder     = (!field.disabled) ? field.placeholder : nil;
```

Comment:
In Ruby, we'd write this as `(field.disabled) ? nil : field.placeholder` so as to avoid dealing with negatives (the brain just isn't wired that way). Is this the convention in Swift, though?